applications/Cyborg.md

```diff
@@ -0,0 +1,178 @@
+
+Second, the unparalleled ecosystem and technical support provided by Polkadot and Kusama have been instrumental in our development, and we've had the privilege of closely collaborating with them over the years.
+
+Lastly, the substrate framework stands out as the premier tool for crafting blockchains from the ground up in today's landscape. Our commitment to excellence aligns with the capabilities it offers.
+
+### Project Details
+
+#### Architecture:
+
+The core of our computing solution relies on both off-chain components and substrate pallets that work harmoniously with each other.
+
+Cyborg is a fusion of a blockchain network and numerous computing clusters, collaborating to offer an effective decentralized solution for verifiable off-chain computing.
+
+Our initial technological goal is to develop a system capable of validating computational outcomes from one off-chain cluster by re-executing them in another using pallet logic.
+
+![photo_2024-02-14_23-43-17](https://github.com/Cyborg-Network/Grants-Program/assets/93442895/d50e7582-8f0f-461a-8668-10a828089404)
+
+The Substrate blockchain primarily serves as the framework for deploying containered tasks and facilitating smart contracts related to billing functions.
+
+Once the docker container is deployed, its workload will be protected by software based measures to prevent data leakage.
+
+It seamlessly integrates with the current cloud-native toolchain for developers, typically avoiding the need for code changes.
+ 
+Developers can monitor details such as resource utilization and application health status through the Cyborg Connect application dashboard.
+
+![dashboard](https://github.com/Cyborg-Network/Grants-Program/assets/93442895/90f160c6-5cef-408b-8db2-c3e550fdb6a8)
+
+### Work Verification System
+
+For this grant application, we are developing a decentralized task verification mechanism to incorporate an on-chain validation system for containerized tasks executed in an off-chain environment. The main components will include the Cyborg AppChain, which will be deployed on Tanssi's Dancebox testnet, the client interface (K3s worker system), an Oracle, and a testing suite.
+
+<img alt="Screenshot 2024-02-15 at 1 09 56 PM" src="https://github.com/Cyborg-Network/Grants-Program/assets/93442895/add65d92-0bf6-40df-a3fb-715fc0efce54">
+
+The objective is to upload a Docker config file along with a publicly available dataset such as IPFS and deploy the container over the IPFS dataset inside one of the nodes of the K3s cluster. The executed computation result will be pushed to the Oracle, which will then feed it to the Cyborg substrate chain. Subsequently, the verification pallet will initiate task execution on another connected cluster to reverify the result. If the initial result and verified result are the same, the chain will approve the computation and add it to the block.
```

Comment:
what happens if they are not matching?

Reply:
Thank you so much for this. I have now updated the statement with a detailed worklflow for this PoC.